docs/markdown/Reference-manual.md

```diff
@@ -855,6 +855,8 @@ This function prints its argument to stdout.
 
 This function prints its argument to stdout prefixed with WARNING:.
 
+*Added 0.43.1*
```

Comment:
Should be 0.44.0?

Reply:
Not sure. Are the point releases only for bugfixes?

Reply:
Yeah, it should be 0.44.0. Updated.